### PR TITLE
Bootstrap git repo

### PR DESCRIPTION
**Pull request by bigkevmcd**
**What type of PR is this?**

**What does this PR do / why we need it**:

It triggers automatic creation of the Repository at the end of the bootstrap process.

**Have you updated the necessary documentation?**

```
$ kam bootstrap
  
Checking dependencies

 ✓  Checking if Sealed Secrets is installed with the default configuration [15ms]
 ✓  Checking if ArgoCD Operator is installed with the default configuration [30ms]
 ✓  Checking if OpenShift Pipelines Operator is installed with the default configuration [34ms]
  
Starting interactive prompt

? Provide the URL for your GitOps repository https://github.com/bigkevmcd/new-gitops.git
? Select type of image repository External Registry
? Image repository of the form <registry>/<username>/<repository> which is used to push newly built images. quay.io/kmcdermo/testing
? Path to config.json which authenticates image pushes to the desired image registry ~/Downloads/auth.json
? Provide a secret (minimum 16 characters) that we can use to authenticate incoming hooks from your Git hosting service for the Service repository. (if not provided, it will be auto-generated) 
? Provide the URL for your Service repository e.g. https://github.com/organisation/service.git https://github.com/bigkevmcd/go-demo.git
? Is this repository a private repository? yes
? Please provide a token used to authenticate requests to "https://github.com/bigkevmcd/go-demo.git" [? for help] ****************************************
? Provide a secret (minimum 16 characters) that we can use to authenticate incoming hooks from your Git hosting service for the Service repository. (if not provided, it will be auto-generated) 
? Do you want to enable commit-status-tracker? yes
? Add a prefix to the environment names(dev, stage, cicd etc.) to distinguish and identify individual environments? 
? Provide a path to write GitOps resources? .
  
Completing Bootstrap process

 ✓  Authentication tokens encrypted in secrets
 ✓  Pipelines tracker has been configured
 ✓  OpenShift Pipelines resources created
 ✓  Openshift Route for EventListener created
 ✓  Created dev, stage and CICD environments
 ✓  Created repository
 ✓  Bootstrapped OpenShift resources sucessfully

 Next Steps:
 Please refer to https://github.com/redhat-developer/kam/tree/master/docs to get started.
```

* [x] Documentation update is required by this PR.
* [x] Documentation has been updated.

**Which issue(s) this PR fixes**:

Fixes #?

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

File: pkg/pipelines/utils.go [hunk 1]
```diff
@@ -1,10 +1,60 @@
...
+		Namespace:   org,
+		Name:        repoName,
+	}
+	_, _, err = client.Repositories.Create(context.Background(), ri)
+	if err != nil {
+		return fmt.Errorf("failed to create repository %q in namespace %q: %w", repoName, org, err)
+	}
+	return err
+}
+
```
Comment (wtam2018) [inline pkg/pipelines/utils.go:57]: We create an empty repository (un-populated with generated manifests) currently, right?

Comment (bigkevmcd) [reply pkg/pipelines/utils.go:57]: yes, there's a separate story for pushing the resources to the repository.

**Review by wtam2018 — COMMENTED** [review]
/lgtm

**Pull request merged**